engines/e_dasync.c

```diff
@@ -829,7 +833,7 @@ static int dasync_rsa_paramgen_init(EVP_PKEY_CTX *ctx)
 
     if (pparamgen_init == NULL)
         EVP_PKEY_meth_get_paramgen(dasync_rsa_orig, &pparamgen_init, NULL);
-    return pparamgen_init(ctx);
+    return pparamgen_init ? pparamgen_init(ctx) : 1;
```

Comment:
``` C
    return pparamgen_init != NULL ? pparamgen_init(ctx) : 1;
```

Reply:
In all other case too

Reply:
Fixed. Thanks!

Reply:
Did you push your fixes? I don't see them?

Reply:
Oh :(
Sorry

Reply:
Done